radis/lbl/base.py

```diff
@@ -1463,48 +1464,8 @@ def _calc_noneq_parameters(
         # Initializing a cache file for lookup later
         filename = (
             splitext(self.params.dbpath.split(",", 1)[0])[0].rsplit("/", 1)[0]
-            + "/ORIGINAL_FILE_new_columns"
+            + "/ORIGINAL_FILE_{molecule}_{iso}_EvibErot"
```

Comment:
there shouldnt be a hardcoded "ORIGINAL_FILE" here 
also no need for "molecule", "iso", it's already in the path
Just name it "{database_file}_extra_params.hdf5"

Reply:
The `molecule` and `iso` actually aren't specified in my linedb path, which is why I added them in the name. Also since we're saving different files for different isotopes, I think we would need isotope number to be mentioned

for instance, on my device, the path for hitemp databases is `/Users/sagarchotalia/.radisdb/hitemp/*`

Reply:
Sure, but a path name would be for instance `/Users/sagarchotalia/.radisdb/hitemp/CO-05_HITEMP2019.hdf5`, and therefore your cache file should become  `/Users/sagarchotalia/.radisdb/hitemp/CO-05_HITEMP2019_extra_columns.hdf5`

Reply:
Since I’m getting the file name from `self.params.dbpath`, it’s going to remain the same for all isotopes, if I don’t insert the isotope/molecule name in it, so maybe I can go ahead with `/Users/sagarchotalia/.radisdb/hitemp/CO-05_HITEMP2019_<iso>_extra_columns.hdf5`.

Reply:
how do you handle a spectrum with multiple isotopes ? Do you concatenate multiple extra_columns files ? DataFrame is sorted by wavenumber, how can you match the lines?

Reply:
I don't think we can concatenate multiple files, we could insert the required rows as per the number of isotopes included in the calculations. I'll save the required columns into a single file then. But that may result in less accurate spectra. I'll try this out and test it

Reply:
What do you mean by "less accurate spectra"?
If there is the slightest chance the lines are mixed we won't push it. And keep on testing it.

You could add isotopes in your file metadata. And regenerate if it changes.

Users do not change which isotopes are used very often. Worst case scenario , energies will be regenerated once.

Reply:
Okay, got it sir!